src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -125,8 +125,10 @@ private void UserSettingsService_OnSettingChangedEvent(object? sender, SettingCh
 		}
 
 		private void HorizontalMultitaskingControl_Loaded(object sender, RoutedEventArgs e)
+			=> TabControl.TabControlAreaSizeChanged += TabControlArea_SizeChanged;
+
+		private void TabControlArea_SizeChanged(object sender, SizeChangedEventArgs e)
```

Comment:
I don't think you should hook size event in here. This method hooks CurrentInstanceChanged every time the size changed. I think loaded event was good.

Reply:
Yea,  but I've already committed to a more elegant solution.